frontend/packages/dev-console/src/components/QuickStartTile.tsx

```diff
@@ -16,32 +17,62 @@ import {
   Button,
 } from '@patternfly/react-core';
 import { ArrowRightIcon } from '@patternfly/react-icons';
-import { QuickStart } from '@console/app/src/components/quick-starts/utils/quick-start-types';
+import { RootState } from '@console/internal/redux';
+import {
+  QuickStart,
+  AllQuickStartStates,
+  QuickStartStatus,
+} from '@console/app/src/components/quick-starts/utils/quick-start-types';
+import {
+  getActiveQuickStartID,
+  getAllQuickStartStates,
+} from '@console/app/src/redux/reducers/quick-start-reducer';
+import { getActivePerspective } from '@console/internal/reducers/ui';
 import * as QuickStartActions from '@console/app/src/redux/actions/quick-start-actions';
+import { getQuickStartStatus } from '@console/app/src/components/quick-starts/utils/quick-start-utils';
+import { HIDE_QUICK_START_DASHBOARD_TILE_STORAGE_KEY } from '@console/internal/components/dashboard/dashboards-page/cluster-dashboard/cluster-dashboard';
 
 import './QuickStartTile.scss';
 
 export const HIDE_QUICK_START_ADD_TILE_STORAGE_KEY = 'bridge/hide-quick-start-add-tile';
 
 type QuickStartTileProps = {
   quickStarts: QuickStart[];
+  onRemoveTile?: () => void;
+};
+
+type StateProps = {
+  perspective?: string;
+  activeQuickStartID?: string;
+  allQuickStartStates?: AllQuickStartStates;
 };
 
 type DispatchProps = {
   setActiveQuickStart?: (quickStartID: string, totalTasks: number) => void;
 };
 
-type Props = QuickStartTileProps & DispatchProps;
+type Props = QuickStartTileProps & DispatchProps & StateProps;
 
-const QuickStartTile: React.FC<Props> = ({ setActiveQuickStart, quickStarts }) => {
-  const isQuickStartTileHidden =
-    localStorage.getItem(HIDE_QUICK_START_ADD_TILE_STORAGE_KEY) === 'true';
+const QuickStartTile: React.FC<Props> = ({
+  setActiveQuickStart,
+  quickStarts,
+  allQuickStartStates,
+  perspective,
+  onRemoveTile,
+}) => {
+  const { t } = useTranslation();
+  const storageKey =
+    perspective === 'dev'
```

Comment:
You don't need to check for perspective like this. Better to just send the storage_key prop from the parent component. That way this tile can be re-used in any number of places in any perspective.